docs/source/containerization/kubernetes.mdx

```diff
@@ -1,264 +1,270 @@
+To support your different deployment configurations for different environments (development, staging, production, etc.), Apollo recommends separating your configuration values into separate files:
 
-If you had a router running on port 8088 on your localhost, you could exercise the health endpoint as follows:
+- A **common** file, which contains values that apply across all environments.
+- A unique **environment** file per environment, which includes and overrides the values from the common chart while adding new environment-specific values.
+
+In a `helm install` command, each `--values <values-file>` option is applied in the order within the command. Therefore, a common file must be set before an environment file in the `helm install` command, so that the environment file's values are applied last and override the commond file's values as intended. 
+
+For example, deploy with a `common_values.yaml` file applied first and then a `prod_values.yaml` file:
 
 ```bash
-curl "http://localhost:8088/health"
-{"status":"UP"}
+helm install --namespace <router-namespace> --set managedFederation.apiKey="<graph-api-key>" --set managedFederation.graphRef="<graph-ref>"  oci://ghcr.io/apollographql/helm-charts/router --version <router-version> --values router/values.yaml  --values common_values.yaml --values prod_values.yaml
 ```
 
-If you had a router running on your localhost, with default health-check configuration, you could exercise the health endpoint as follows:
+## Configure for migration from gateway
 
-curl "http://localhost:8088/health"
+When [migrating from `@apollo/gateway` to the Apollo Router](../migrating-from-gateway), consider the following tips to maximize the compatibility of your router deployment.
 
-## Using `istio` with the router
+* The gateway's maximum supported request size is 20MB, while the router's default maximum request size is less. For compatibility, set the deployed router's maximium request size to 20MB:
 
-The [istio service mesh](https://istio.io/) is a very popular choice for enhanced traffic routing within Kubernetes.
+  ```yaml title="values.yaml"
+  router:
+    configuration:
+      limits:
+        experimental_http_max_request_bytes: 20000000 #20MB
+  ```
 
-`istio-proxy` pod injection can cause an [issue](https://github.com/apollographql/router/issues/3533) in the router. The router may start executing at the same time that istio is reconfiguring networking for the router pod. This is an issue with `istio`, not the router, and you can resolve it by following the advice in [istio's injection documentation](https://istio.io/latest/docs/ops/common-problems/injection/#pod-or-containers-start-with-network-issues-if-istio-proxy-is-not-ready).
+* The router's timeout is increased to accommodate slow subgraphs. Only HTTP 1.1 suport is enabled.
+
+```yaml title="values.yaml"
+router:
+  configuration:
+    traffic_shaping:
+      router:
+        timeout: 6min
+      all:
+        experimental_enable_http2: false
```

Comment:
Remove line 259. We aren't using that in-house. It was a short lived experiment.